pilot/cmd/pilot-agent/main.go

```diff
@@ -329,7 +342,19 @@ func getPilotSAN(domain string, ns string) []string {
 			}
 		}
 		spiffe.SetTrustDomain(pilotTrustDomain)
-		pilotSAN = append(pilotSAN, envoy.GetPilotSAN(ns))
+	}
+
+}
+
+func getPilotSAN(ns string) []string {
```

Comment:
Similarly, can we have getMixerSAN?

Reply:
unified both mixer and pilot san into one parameterized func